global.json

```diff
@@ -238,15 +262,57 @@
     }, {
       "id": "ios_project",
       "priority": 0.84,
+      "name": "Native iOS Blank Project",
+      "description": "A FeedHenry blank project with one iOS Client App and a cloud instance with a single endpoint",
+      "type": "default",
+      "icon": "icon-apple",
+      "category": "Sample Projects",
+      "screenshots": [{
+        "url": "https://raw.githubusercontent.com/feedhenry-templates/blank-ios-app/master/screenshots/iosproject_container.png"
+      }],
+      "appTemplates": ["blank_native_ios_client", "hello_world_mbaas_instance"]
+    },{
+      "id": "ios_blank_cocoapods_project",
```

Comment:
have tested this locally and on a single node instance (http://testing.jmadigan.skunkhenry.com) - built okay